### ISSUE TITLE
MiniTest Test Runner Exits After One Test Failure

### ISSUE THREAD

**Issue by lcreid** [bootstrap-ruby/rails-bootstrap-forms#371]
When running tests, MiniTest exists after one test failure. For example:
```
vagrant@rails-5-1:/vagrant$ bundle exec rake
Run options: --seed 41303

# Running:

............................................F

Failure:
BootstrapCheckboxTest#test_check_box_is_wrapped_correctly [/vagrant/test/bootstrap_checkbox_test.rb:12]:
-<div aclass="form-check">
+<div class="form-check">
   <label class="form-check-label" for="user_terms"><input name="user[terms]" type="hidden" value="0"/><input class="form-check-input" id="user_terms" name="user[terms]" type="checkbox" value="1"/> I agree to the terms</label>
 </div>
\ No newline at end of file



/var/lib/gems/2.3.0/gems/railties-5.1.4/lib/rails/test_unit/reporter.rb:70:in `method': undefined method `test_check_box_is_wrapped_correctly' for class `Minitest::Result' (NameError)
	from /var/lib/gems/2.3.0/gems/railties-5.1.4/lib/rails/test_unit/reporter.rb:70:in `format_rerun_snippet'
	from /var/lib/gems/2.3.0/gems/railties-5.1.4/lib/rails/test_unit/reporter.rb:23:in `record'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:786:in `block in record'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:785:in `each'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:785:in `record'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:334:in `run_one_method'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:321:in `block (2 levels) in run'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:320:in `each'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:320:in `block in run'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:360:in `on_signal'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:347:in `with_info_handler'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:319:in `run'
	from /var/lib/gems/2.3.0/gems/railties-5.1.4/lib/rails/test_unit/line_filtering.rb:9:in `run'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:159:in `block in __run'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:159:in `map'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:159:in `__run'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:136:in `run'
	from /var/lib/gems/2.3.0/gems/minitest-5.11.1/lib/minitest.rb:63:in `block in autorun'
rake aborted!
Command failed with status (1)
/var/lib/gems/2.3.0/gems/rake-12.3.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => default => test
(See full trace by running task with --trace)
```

This may have started after merging #366, but I can't say for sure. I can reproduce the problem using a clean Vagrant box running Ubuntu 16.04. Is anyone else seeing this?

**Comment by mattbrictson** [comment]
Minitest 5.11 changed its reporter API and it broke projects like Rails that use custom reporters. Immediate workaround is to pin minitest at `~> 5.10.3`. Rails has been updated to work with the new API but it is unclear whether this will be backported to Rails < 5.2.

**Comment by lcreid** [comment]
Thanks! Apologies for being lazy. I did one Google and nothing popped up. Must have used the wrong words. 

Should I submit a PR to pin minitest at 5.10.3 for now?

**Comment by mattbrictson** [comment]
> Should I submit a PR to pin minitest at 5.10.3 for now?

👍

**Comment by donv** [comment]
This one is resolved by #372 , I think.
